spec/helpers/session.rb

```diff
@@ -0,0 +1,31 @@
+module SessionHelpers
+  def sign_up(user)
+    visit '/users/sign_up'
+    fill_in :user_username, with: user.username
+    fill_in :Email, with: user.email
+    fill_in :user_password, with: user.password
+    fill_in :user_password_confirmation, with: user.password_confirmation
+    click_button 'Sign up'
+  end
+
+  def sign_in(email:, password:)
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `email`. You can also write as `sign_in(*)` if you want the method to accept any arguments but don't care about them. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)<br>Lint/UnusedMethodArgument: Unused method argument - `password`. You can also write as `sign_in(*)` if you want the method to accept any arguments but don't care about them. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)